### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by SnazzyCoder**
🚀 Made the project readme more descriptive
- Added important setup guidelines
- Added Known issues section
- Converted h3 to h1 at the top level (semanticized markup)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -32,4 +36,57 @@ A [seedr.cc](https://www.seedr.cc/) inspired web application to download torrent
...
+This section tells you how to setup a self-hosted instance of Autolycus
+
+## Pre-requisites
+
+You will need the following before installation:
+
+1. Docker Command line tools
+    1. [Docker Engine](https://docs.docker.com/engine/install)
+    1. [Docker Compose](https://docs.docker.com/compose/install/) command
+1. Postgresql database with its username, password and host
```
Comment (imneonizer) [inline Readme.md:52]: Users don't need to have already running Postgres instance, `docker-compose` will take care of it.

Comment (SnazzyCoder) [reply Readme.md:52]: Oh...

Comment (imneonizer) [reply Readme.md:52]: Please remove this line
````
Postgresql database with its username, password and host
````

**Review by SnazzyCoder — COMMENTED** [review]
PostgreSQL removed from dependencies. As requested 😃

**Comment by imneonizer** [comment]
Hey can anyone explain how there are two different accounts pushing to the same fork.
This PR is about updating the `Readme` but now I see commits which is fixing css.

There are 5 files changed during the commits especially `package-lock.json` and `yarn.lock`  and have significant additions related to `node_modules`

I am trying to keep the bundle size small.
I will request you to open another PR with only `Readme` updated.

**Pull request closed without merging**